plugins/modules/network_compliance_workflow_manager.py

```diff
@@ -1261,7 +1030,7 @@ def get_task_result(self, task_id, device_ids):
         start_time = time.time()
 
         while True:
-            response = self.get_task_status(task_id, task_name)
+            response = self.get_task_status_from_tasks_by_id(task_id, task_name)
```

Comment:
In the latest dnac.py, it is defined as:
 def get_task_status_from_tasks_by_id(self, task_id, task_name, success_msg):

Can you check?

Reply:
Based on the def get_task_status_from_tasks_by_id(self, task_id, task_name, success_msg) function, the get_task_result function in the network_compliance_module has been modified.